### DISCUSSION TITLE
Please help me troubleshoot why my LEDs are not working.

### DISCUSSION THREAD

**Discussion by finiteuser** [iNavFlight/inav#9682] [Q&A]
I installed 4x [Matek 2182-8Slim LED strips](https://www.mateksys.com/?portfolio=2812-8slim). According to my multimeter I don't have any shorts, and I have good connectivity between all the grounds. And good connectivity between all the 5Vs.  The D-out pads are connected to the D-in pads on the next strip, with no shorts detected between them. 

![config1](https://github.com/iNavFlight/inav/assets/145856092/fa6d1d6c-058b-4048-9222-e1bca9cd92ec)
![config2](https://github.com/iNavFlight/inav/assets/145856092/230660ba-c4fc-4b8b-a187-0237684aed95)
![wiring](https://github.com/iNavFlight/inav/assets/145856092/fdbf1659-246a-44b4-83b4-8851a12e1b44)
![IMG_0514](https://github.com/iNavFlight/inav/assets/145856092/fc0610aa-1bbd-47ae-a168-e77d0e9075ec)

I tested the voltage with the battery plugged in, and each strip is seeing 5V.  I thought I enabled everything correctly in the configurator, but this is the first time I have tried to install LEDs. It's completely possible that I have missed something.  Are they set to only turn on when the quad is armed? I'm indoors and haven't gotten GPS lock. The wiki page at https://github.com/iNavFlight/inav/blob/master/docs/LedStrip.md seemed to indicate that with this configuration, the LEDs should just be "on," at all times. But it's possible I misunderstood.  I don't know what I'm doing.

I typed "LED" in the CLI and this is what it returned.

# LEDs
led 0 0,0::C:1
led 1 1,0::C:1
led 2 2,0::C:1
led 3 3,0::C:1
led 4 4,0::C:1
led 5 5,0::C:1
led 6 6,0::C:1
led 7 7,0::C:1
led 8 0,1::C:1
led 9 1,1::C:1
led 10 2,1::C:1
led 11 3,1::C:1
led 12 4,1::C:1
led 13 5,1::C:1
led 14 6,1::C:1
led 15 7,1::C:1
led 16 0,2::C:6
led 17 1,2::C:6
led 18 2,2::C:6
led 19 3,2::C:6
led 20 4,2::C:6
led 21 5,2::C:6
led 22 6,2::C:6
led 23 7,2::C:6
led 24 0,3::C:2
led 25 1,3::C:2
led 26 2,3::C:2
led 27 3,3::C:2
led 28 4,3::C:2
led 29 5,3::C:2
led 30 6,3::C:2
led 31 7,3::C:2

# LED color

# LED mode_color

Is there an error message I could look for? What's going on?

**Comment by finiteuser** [comment]
Status update:

Re-reading the wiki section on how current draw for LEDs can be high, I installed a 5V UBEC thinking that might solve the problem. It did not.  The LEDs still don't light up. Can anyone help me? Now that the LED circuit is on a BEC I don't know what else to do.

**Reply by QuocSiPHU** [reply]
In the configurator you must start setting up your led from led number 0!
However in the CLI it's ok!

**Comment by dj0abr** [comment]
any updated information about this issue?
I have the same problem.
Configured the LED strips in Betaflight, all works well.
Flashed INAV and tried the same, all LEDs dark.

**Reply by finiteuser** [reply]
It's still not working.
Updating to the latest version of inav and flashing the latest firmware did not resolve the issue.
Interesting that it works in betaflight. Suggests the problem is not hardware related.
